src/_base/harness/attributes/common.yml

```diff
@@ -118,6 +118,11 @@ attributes.default:
       when: -f "package.json"
       steps:
         - run "npm install"
+        - |
+          if [ "$APP_BUILD" == "static" ]; then
+            run "rm -rf node_modules/"
+            run "npm cache clean --force"
+          fi
```

Comment:
hmm, do we need a assumption that this is changed or should we add a `passthru "npm build"` assumption?

Reply:
Yeah was wondering if `npm run build` should be a convention, though this will cause it to fail if not present in package.json

Reply:
I think yarn is more convention as well

Reply:
Updated to use yarn :+1:

Reply:
I did switch to yarn, but reading our engineering guidelines we favour npm. I'll switch back to npm.